xlive/Blam/Engine/game/players.h

```diff
@@ -81,6 +81,16 @@ struct s_player_profile_traits
 };
 ASSERT_STRUCT_SIZE(s_player_profile_traits, 16);
 
+#pragma pack(push, 1)
+struct s_player_identifier
+{
+	// same as XUID
+	DWORD unk1;
```

Comment:
DWORD -> uint32

Reply:
why not just a single uint64?

Reply:
iirc this struct actually splits it into two values